src/DoctrineORMModule/Module.php

```diff
@@ -139,7 +138,14 @@ public function initializeConsole(EventInterface $event)
         $entityManager = $serviceLocator->get('doctrine.entitymanager.orm_default');
         $helperSet     = $cli->getHelperSet();
 
-        $helperSet->set(new QuestionHelper(), 'question');
+        if (class_exists('Symfony\Component\Console\Helper\QuestionHelper')) {
+            $helperSet->set(new \Symfony\Component\Console\Helper\QuestionHelper(), 'question');
+        }
+
+        if (class_exists('Symfony\Component\Console\Helper\DialogHelper')) {
+            $helperSet->set(new \Symfony\Component\Console\Helper\DialogHelper(), 'dialog');
```

Comment:
Ah.. i see this was added in symfony/console ~2.7. I could call the constructor with false to avoid the deprecation warning, but this would be an error in PHP 7 with symfony/console < 2.7, because of the strict argument count check added in PHP 7.

@stof Do you think this is an issue? If yes i would need to check with ReflectionFunctionAbstract::getNumberOfParameters or similar for the signature of the constructor. If no, i would just change the code to new DialogHelper(false)...